Solutions/Trend Micro Apex One/Data Connectors/TrendMicro_ApexOne.json

```diff
@@ -2,8 +2,8 @@
     "id": "TrendMicroApexOne",
```

Comment:
Please rename all occurrences in this file `Data Connectors/TrendMicro_ApexOne.json` of `Azure Sentinel` to `Microsoft Sentinel`